scheds/rust/scx_lavd/meson.build

```diff
@@ -1,7 +1,8 @@
-custom_target('scx_lavd',
+sched = custom_target('scx_lavd',
               output: '@PLAINNAME@.__PHONY__',
               input: 'Cargo.toml',
               command: [cargo, 'build', '--manifest-path=@INPUT@', '--target-dir=@OUTDIR@',
                         cargo_build_args],
               env: cargo_env,
-              build_by_default: true)
+              depends: [sched],
```

Comment:
Is the dependency being different from other schedulers intentional?

Reply:
Ah, right. It wasn't there from before. I don't think that's correct but a separate issue.